site/app/templates/admin/admin_gradeable/rubric/ElectronicFileRubric.twig

```diff
@@ -9,6 +9,21 @@
   <option value="5" {{ gradeable.getPrecision() == 5 ? 'selected' : '' }}>5</option>
   <option value="10" {{ gradeable.getPrecision() == 10 ? 'selected' : '' }}>10</option>
 </select>
+<br>
+<br>
+<fieldset>
+    <legend>
+    Will graders be allowed to use custom marks?
```

Comment:
Let's only show this red message if there already are custom marks and the instructor is attempting to disable them.
(Never show the red message if the "yes" radio button is selected.)